examples/msgRenewLock.ts

```diff
@@ -0,0 +1,40 @@
+import { OnMessage, OnError, MessagingError, delay, ServiceBusMessage, ReceiveMode, Namespace } from "../lib";
+import * as dotenv from "dotenv";
+dotenv.config();
+
+const connectionString = "SERVICEBUS_CONNECTION_STRING";
+const entityPath = "QUEUE_NAME";
+const str = process.env[connectionString] || "";
+const path = process.env[entityPath] || "";
+console.log("str: ", str);
+console.log("path: ", path);
+
+let ns: Namespace;
+async function main(): Promise<void> {
+  ns = Namespace.createFromConnectionString(str);
+  const client = ns.createQueueClient(path, { receiveMode: ReceiveMode.peekLock });
+  const onMessage: OnMessage = async (brokeredMessage: ServiceBusMessage) => {
+    console.log(">>> Message: ", brokeredMessage);
+    console.log("### Actual message:", brokeredMessage.body ? brokeredMessage.body.toString() : null);
+    const result = await client.renewLock(brokeredMessage.lockToken!);
```

Comment:
does it make sense for the user to actually provide the lock token or to just provide the message itself? that way they wouldn't have to worry about the implementation details of how locking works

Reply:
it also allows you to set the `lockedUntilUtc` for them rather than them setting it outside of the call

Reply:
Nevermind i see what you're doing. I'd probably have the example show passing the full message because i think that's likely to be the more common pattern

Reply:
The method takes a union of lockToken | Message. Yup, shall update the example.